.github/workflows/python-app.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ "master" ]
+    paths: ['backend/**', '.github/**']
+  pull_request:
+    branches: [ "master" ]
+    paths: ['backend/**', '.github/**']
+
+permissions:
+  contents: read
+
+env:
+  DAY_OF_WEEK: Monday
+  DB_SERVER: localhost
+  DB_PORT: 3306
+  DB_USER: test
+  DB_PASSWORD: test
+  DATABASE: test
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./backend
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
```

Comment:
Can we not rely on `requirements.txt` to supply the flake8 and pytest versions? Also, pre-commits rely on black and not flake8, I don't know if that will cause us to have different linting mechanisms